### ISSUE TITLE
How to activate tracking of additional extensions?

### ISSUE THREAD

**Issue by gneumann-hh** [eaxs/User-Activity#3]
In configuration of Module User Activity, All Extensions is selected. Unfortunately, tracking list will show only activities concerning articles, is there a way to track also additional extensions (e.g. jdownloads, JEvents, JoomGallery ...) .  If yes, how to do?

**Comment by kyleledbetter** [comment]
Yes, you'll need additional plugins for each additional extension. We include 2 example plugins (site and admin tracking) for com_content. You'd follow the same methods used https://github.com/eaxs/User-Activity/tree/master/plugins

**Comment by derChristoph** [comment]
Is there a way (Plugin) to track everything that is done in the Backend?
